portal/config/eproms/Questionnaire.json

```diff
@@ -5851,6 +5851,42 @@
                 "display": "Other",
                 "code": "irondemog_v3.26.8"
               }
+            },
+            {
+              "valueCoding": {
+                "display": "African",
```

Comment:
@ivan-c  I'm removing this "African" option from the AE, as the only sites that use it changed their mind and no longer want it (https://movember.atlassian.net/browse/TN-3285). I expect that I should leave it here, since it has been live for some months and we likely have some data for it. Is that the case?

Reply:
Thanks @ivan-c  Reading that, I'm wondering if I shouldn't have removed the option from the AE a few weeks ago https://gitlab.cirg.washington.edu/svn/dhair2/-/merge_requests/838/diffs , but you did approve that. Should I restore that (with a condition to either always exclude it, or set sequence = 0)?